app/services/host_aspects.rb

```diff
@@ -0,0 +1,11 @@
+module HostAspects
```

Comment:
How about using the singular - `Aspect`? The module is not implementing multiple aspects as far as I can see.